zjit/src/hir.rs

```diff
@@ -2682,37 +2682,47 @@ pub fn iseq_to_hir(iseq: *const rb_iseq_t) -> Result<Function, ParseError> {
                     break;  // Don't enqueue the next block as a successor
                 }
                 YARVINSN_getlocal_WC_0 => {
-                    // TODO(alan): This implementation doesn't read from EP, so will miss writes
```

Comment:
Please add some HIR parsing tests

Reply:
Could you give me some example code to do so? I'm not sure how to extract an ISEQ pointer (or `RubyVM::InstructionSequence`) out of an eval, which has `ISEQ_TYPE_EVAL`.